packages/web/app/src/components/organization/members/permission-selector.tsx

```diff
@@ -33,6 +33,7 @@ export const PermissionSelector_PermissionGroupsFragment = graphql(`
       title
       isReadOnly
       warning
+      isAssignableByViewer
```

Comment:
This is new field indicates whether a user can assign the permission, it is only relevant in the context of PATs tho.